File: diretorio3/Retangulo.java
//atividade 30/08/24
//aluno: João Pedro Araújo Inácio

public class Retangulo {
    private double comprimento;
    private double largura;

    public Retangulo(double comprimento, double largura) {
        this.comprimento = comprimento;
        this.largura = largura;
    }

    public double calcularArea() {
        return comprimento * largura;
    }

    public double calcularPerimetro() {
        return 2 * (comprimento + largura);
    }

    public static void main(String[] args) {

        Retangulo ret = new Retangulo(200, 300);
        System.out.println("area obtida: " + ret.calcularArea());
        System.out.println("peri­metro obtido: " + ret.calcularPerimetro());
    }
}

File: diretorio3/Retangulo.go
//atividade 30/08/24
//aluno: João Pedro Araújo Inácio

package main

import "fmt"

type Retangulo struct {
	comprimento, largura float64
}

func (r Retangulo) calcularArea() float64 {
	return r.comprimento * r.largura
}

func (r Retangulo) calcularPerimetro() float64 {
	return 2 * (r.comprimento + r.largura)
}

func main() {
	ret := Retangulo{comprimento: 200, largura: 300}
	fmt.Printf("area obtida: %.2f\n", ret.calcularArea())
	fmt.Printf("perimetro obtido: %.2f\n", ret.calcularPerimetro())
}
